Documentation/kubernetes-on-baremetal.md

```diff
@@ -1,13 +1,34 @@
 # Kubernetes Installation on Bare Metal &amp; CoreOS
 
-This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS.
-After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
+This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS. After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
 
-## Deployment Prerequisites
+## Deployment Requirements
```

Comment:
Why this specific one and not the other `##` headings? Doesn't it look better with capitalized headings? I haven't seen folks doing it this way and I'm curious.